downstream/modules/eda/proc-eda-create-event-stream.adoc

```diff
@@ -0,0 +1,48 @@
+[id="eda-create-event-stream"] 
+
+= Creating an event stream
+
+You can create event streams that will be attached to a rulebook activation. 
+
+.Prerequisites
+
+* If you will be attaching your event stream to a rulebook activation, ensure that your activation has a decision environment and project already set up.
```

Comment:
If you attach your event stream... you must ensure...

Reply:
I'm going to forego this suggestion. I think this is okay. Screenshot of IBM style attached.
![Screenshot from 2024-09-27 10-49-04](https://github.com/user-attachments/assets/1c53c018-4d71-43a7-a485-41d83891f883)